### ISSUE TITLE
Keyboard shortcuts within find and replace to toggle ignore caps, regex, and only in cell replace?

### ISSUE THREAD

**Issue by excetara2** [jupyter/notebook#1190]
So within find and replace there are three options that can be toggled. Can these be toggled with a keyboard shortcut? If not can this feature be added or is there a way to add in some custom javascript to enable it?

**Comment by Carreau** [comment]
Likely possible but not too easy. 

I would suggest using Shift-Tab/Space to toggle the button. The  search field should be automatically refocused.

**Comment by excetara2** [comment]
Yes that works just fine for what I need. I just thought it might be easy
with all the other shortcuts already implemented and it being in its own
pop-up box. But of note when you toggle the all code blocks or just the
selected code blocks with space it seems to send you to the replace instead
of the seach. The other two send you to replace but maybe because when you
toggle that it runs a check even if the fiels are empty and says nothing is
matching. Anyways tahnks for the help.

**Comment by Carreau** [comment]
No problem, one thing that would be helpful is try to think which shortcut for what action.

**Comment by excetara2** [comment]
Yeah that is tough as I think many things will interfere with  the built-in functions of the browser. Maybe the F keys could be used or ctrl+. for blocks. Then ctrl+[ and ctrl+] for case-matching and regex( assume that's the middle option.)
